app/controllers/event_disavowal_controller.rb

```diff
@@ -10,13 +10,13 @@ def new
       success: true,
       extra: EventDisavowal::BuildDisavowedEventAnalyticsAttributes.call(disavowed_event),
     )
-    analytics.event_disavowal(**result.to_h)
+    analytics.event_disavowal(user_id: disavowed_event.user_id, **result.to_h)
```

Comment:
Should we implement this in [`BuildDisavowedEventAnalyticsAttributes`](https://github.com/18F/identity-idp/blob/main/app/services/event_disavowal/build_disavowed_event_analytics_attributes.rb) instead? That service class pattern is not very conventional, but at least it gives us some consistency in how we build the analytics hash, and should require only updating in one place (vs. two here).

Reply:
That seems to work great. 
In regards to above, should user_id be documented in analytics_events? I took it out.

Reply:
I think based on discussion it makes sense to keep the documentation.